example/js/scalableTable.js

```diff
@@ -0,0 +1,840 @@
+angular.module('scalable-table', [])
```

Comment:
@myfoxtail I have used main file now. But I should been change the line 83 to `scope.$on('$destroy', function() {` There is removed `$`, because it caused an error.